app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/BusinessEntityController.scala

```diff
@@ -337,7 +339,7 @@ class BusinessEntityController @Inject()(
     mapping("businessEntityEori" -> mandatory("businessEntityEori"))(FormValues.apply)(FormValues.unapply).verifying(
     "businessEntityEori.regex.error",
     fields => fields match {
-      case a if a.value.matches(EORI.regex) => true
+      case a if s"$eoriPrefix${a.value}".matches(EORI.regex) => true
```

Comment:
You can probably add the prefix in the form itself the reduce the duplication.

Reply:
updated as per comments

Reply:
@sam-miele @PaulThor please let me know if these changes looks ok